sunpycube/cube/cube_utils.py

```diff
@@ -519,16 +517,38 @@ def get_cube_from_sequence(cubesequence, item):
     """
     if isinstance(item, int):
         result = cubesequence.data[item]
-
     if isinstance(item, slice):
-        data = cubesequence.data[item]
-        meta = cubesequence.meta
-        result = cubesequence._new_instance(data, meta)
-
+        data = cubesequence.data[item.start:item.stop]
```

Comment:
Is there a reason you don't you just index with the slice object itself?  That way a user could use the slice step to get e.g. every 2nd cube if they want.